rel/overlay/etc/default.ini

```diff
@@ -299,6 +299,9 @@ verify_ssl_certificates = false
 ;ssl_trusted_certificates_file = /etc/ssl/certs/ca-certificates.crt
 ; Maximum peer certificate depth (must be set even if certificate validation is off).
 ssl_certificate_max_depth = 3
+; improve testing
+start_delay=0
```

Comment:
That shouldn't be here since it harms general logic and actually is a hack to not sleep in tests.